src/components/JournalIndexPage/JournalIndexPage.scss

```diff
@@ -0,0 +1,63 @@
+.journal-card-container {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+}
+
+.journal-card-link {
+  &:hover {
+      text-decoration: none;
+  }
+  margin-left: 0px;
+
+  .journal-card {
+    height: 360px;
+    width: 360px;
+    border-radius: 4px;
+    background-color: $white;
+    border: solid 1px #d6dbe0;
```

Comment:
Would prefer all colors be variables. It seems this one is used here, and in the search views.

Reply:
* can just be a named color like `$deep-sea-blue`; doesn't need to be themeable.